.cascade-code/Chapman.edu/_components/left_nav/left_nav_no_siblings.vtl

```diff
@@ -0,0 +1,171 @@
+## Formerly LeftNavNoSiblings. Primarily used for index pages.
+## New left nav format that works without a block
+
+#set ( $parentDir = $currentPage.parentFolder )
+
+## Builds the left navigation for non-index pages (only shows the title)
+#macro ( buildNonIndexLeftNav )
+    #buildNonIndexLeftNavTitle()
+    ## Needs to have empty <ul> tags because otherwise title does not have a bottom margin
+    ## Without it, other left col widgets will show up directly under the left nav with no space
+    <ul></ul>
+#end
+
+## Builds the left navigation for index pages
+#macro ( buildIndexLeftNav )
+    #buildIndexLeftNavTitle()
+    <ul>
+    #foreach ( $child in $parentDir.children )
+        #buildLeftNavMenu($child)
+    #end
+    </ul>
+#end
+
+## Builds the header for the left nav on index pages
+#macro ( buildIndexLeftNavTitle )
+    <div class="leftTitle">
+        <ul>
+            <li class="title">
+            <a href="${currentPage.link}" target="_top" title="${_EscapeTool.xml($currentPage.label)}">
+                ${_EscapeTool.xml($currentPage.label)}
+            </a>
+            </li>
+        </ul>
+    </div>
+#end
+
+## Builds header for non-index pages that use this format
+## Non-index pages should display the title as the page display name and nothing else
+#macro ( buildNonIndexLeftNavTitle )
```

Comment:
I made two separate macros for the left nav titles, one for index pages and one for non-indexes pages, but if we think it's not really necessary then I can just have one macro that has a conditional in it that checks which kind of page it is

Reply:
I like keeping 2 macros. Even though some repeat code in them it's probably easier to get your brain around them and easier to see differences if they're kept separate.  Another thing in this same section of the code.. You're putting the same verbiage in the title parameter of the menu links as is in the visible link text. I think that is actually worse for accessibility than better. The title is supposed to supplement or give you something different, or if multiple links on page share the same link text but go to different urls then use title to differentiate. I think we probably need to remove OR you can concatenate the users's label with static text such as Click on menu to go to page about ....  I can see this helping if an item is in left nav and a breadcrumb link. I read about on https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text

Reply:
I could be wrong but I *think* that some of the classes in here eg, s-f-idx, c-f and c-p are not really for styling but codes I'd added once so I could look at the html source and see what a link in the menu represented eg I think c-f was child and a folder (index) and c-p was a chid and a page. We can keep or leave but I don't think you'll find any css styles related to them. Was when I was trying to rewrite this leftnav (several times) in the past and trying to sort out which sections of velocity produced which parts of final menu.

Reply:
If these classes are non-functional then I agree: remove them.

Reply:
> You're putting the same verbiage in the title parameter of the menu links as is in the visible link text. I think that is actually worse for accessibility than better. The title is supposed to supplement or give you something different, or if multiple links on page share the same link text but go to different urls then use title to differentiate. I think we probably need to remove OR you can concatenate the users's label with static text such as Click on menu to go to page about...

I talked with Kayla about this in standup this morning. I agree that we should probably fix this. But this is existing behavior so changing it now is beyond the scope of this story. It will have to be written up as a new user story.